controller/search_blackbox_test.go

```diff
@@ -783,13 +796,13 @@ func (s *searchBlackBoxTest) TestSearchQueryScenarioDriven() {
 	s.T().Run("space=spaceID AND (type=bug AND state=resolved AND (assignee=bob OR assignee=alice))", func(t *testing.T) {
```

Comment:
Just curious... Could you parse the T.Name and use that as the 'filter'? :)

Reply:
There is a plan to introduce space_name, iteration_name, area_name etc.